model.py

```diff
@@ -65,7 +65,8 @@ def from_name(cls, name: str):
     "Mistral-7B": dict(n_layer=32, n_head=32, n_local_heads=8, dim=4096, intermediate_size=14336, vocab_size=32000),
     "stories15M": dict(n_layer=6, n_head=6, dim=288),
     "stories110M": dict(n_layer=12, n_head=12, dim=768),
-    "Llama-3-8B": dict(block_size=8192, n_layer=32, n_head=32, n_local_heads=8, dim=4096, intermediate_size=14336, vocab_size=128256),
+    "llama-3-8b": dict(block_size=8192, n_layer=32, n_head=32, n_local_heads=8, dim=4096, intermediate_size=14336, vocab_size=128256),
```

Comment:
Keep as "Llama-3-8B" and "Llama-3-70B" so that the string matching [here](https://github.com/pytorch-labs/gpt-fast/blob/e2cfa34145e7f14f8390c8ab727f8d7451b26fec/model.py#L47) can work out of the box, when using HF folder name of "Meta-Llama-3-8B-Instruct" or "Meta-Llama-3-70B-Instruct"

config = [config for config in transformer_configs if config in str(name).upper() or **config in str(name)**]